app/services/SubmissionService.scala

```diff
@@ -54,10 +54,16 @@ class SubmissionService @Inject()(
       subscriptionsToUpdate => {
         Future.sequence(subscriptionsToUpdate.map {
           case (year, psubs) =>
-            if (isDuplicateInSeqPsubs(psubs)) {Future.failed(SubmissionValidationException("Duplicate Psubs"))}
-            else {professionalBodiesService.validateYearInRange(psubs.map(_.name), year)}
+            if (isDuplicateInSeqPsubs(psubs)) {
```

Comment:
This seems a bit of an odd place to do this check, are there are any thoughts on how we can refactor to get this same goal.